pkg/api/v1/conversion.go

```diff
@@ -380,31 +380,7 @@ func Convert_v1_ReplicationControllerSpec_To_api_ReplicationControllerSpec(in *R
 }
 
 func Convert_api_PodStatusResult_To_v1_PodStatusResult(in *api.PodStatusResult, out *PodStatusResult, s conversion.Scope) error {
-	if err := autoConvert_api_PodStatusResult_To_v1_PodStatusResult(in, out, s); err != nil {
-		return err
-	}
-
-	if old := out.Annotations; old != nil {
```

Comment:
Is it true that this file contains only functions in format
``` 
func Convert_v1_Field_To_api_Field(in *Field, out *api.Field, s conversion.Scope) error 
```
where type of Field is defined in types.go?

Mentioned function would need access to Annotations and InitContainers field, that means pointer to Pod. Is there a file where we put this kind of functions? Is it more acceptable not to modify conversions for init containers promotion?